### PR TITLE
Create the base structure

### PR DESCRIPTION
**Pull request by mpourismaiel**

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -64,7 +64,7 @@ lastmod = ["lastmod", ":git", "date"]
   weight = 10
 
 [[menu.main]]
-  url = "/about"
+  url = "/syna"
```
Comment (stp-ip) [inline config.toml]: Shouldn't this be https://about.okkur.org/syna instead?

File: content/blog/2019-10-29-hello-world/content.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+fragment = "content"
+weight = 100
+
+title = "Hello World!"
+
+[sidebar]
+  active = true
++++
+
+This is the first blog post of the Okkur Labs.
```
Comment (stp-ip) [inline content/blog/2019-10-29-hello-world/content.md]:
```suggestion
This is the first blog post of Okkur Labs.
```

File: layouts/partials/helpers/container.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{{- if .start }}
```
Comment (stp-ip) [inline layouts/partials/helpers/container.html:1]: Why do we need custom a helper here?

File: layouts/partials/fragments/content.html [hunk 4]
```diff
@@ -0,0 +1,64 @@
+{{- $self := .self -}}
```
Comment (stp-ip) [inline layouts/partials/fragments/content.html:1]: Why do we need custom a fragment here?

File: config.toml (hunk 1 above)
Comment (mpourismaiel) [reply config.toml]: I forgot to change it back. Shouldn't we remove the menus? As okkur website will override them. Or am I wrong?

File: layouts/partials/helpers/container.html (hunk 3 above)
Comment (mpourismaiel) [reply layouts/partials/helpers/container.html:1]: I'm gonna patch it in the main repo and remove it from here later on. It needs a bit more work for the theme though. Just wanted it to work as I wanted it to. This is need so the content fragment would support the padding variable and remove the duplicate code for the container used in the content fragment. 

The code change is also done in the content fragment. I'll push the change to Syna ASAP.

File: config.toml (hunk 1 above)
Comment (stp-ip) [reply config.toml]: It won't be merged as far as my thinking goes as it is on a separate subdomain and with a separate netlify process.

**Comment by stp-ip** [comment]
This still needs a working netlify config and removal of non blog things such as the readme.

**Review by stp-ip — APPROVED** [review]

**Pull request merged**